src/fromager/sources.py

```diff
@@ -141,23 +141,35 @@ def default_download_source(
     sdist_server_url: str,
 ) -> tuple[pathlib.Path, str, str]:
     "Download the requirement and return the name of the output path."
-    url_template = ctx.settings.sdist_download_url(req.name)
-    rename_to_template = ctx.settings.sdist_local_filename(req.name)
+    logger.debug(f"{req.name}: loading resolver parameters from settings")
```

Comment:
Well the hope is that it comes right before: https://github.com/python-wheel-build/fromager/pull/207/files#diff-e99a2837a7d36f9f14de5c4b54a1db688c08f148f62b8e3d5cb82937419dbca9R153

I was not able to cleanly combine these two statements hence the separate logs. Another option could be to log `settings._get_package_settings` and making that public

Reply:
This might be a philosophical difference.

I generally want to log before something that may take some time, so if it looks like the program is "stuck" a user can reason about whether maybe it's just taking a while to compute or download something.

Then I also want to log when non-default behaviors are triggered, like an override is being used (or not).

This message is coming just before a lookup that should be very fast, and the results are logged later. So I think we can drop this message entirely and then fix up the phrasing of the other one and the user will have all the info they need.

Reply:
Yep we can drop the first log statement. If we want to fix the phrasing of the second log statement to indicate whether a value came from settings or not, then I will have to add additional checks just for that. It won't be as clean. If we want to indicate if it came from settings then I would prefer to do it they way we are discussing below